config/locales/devise/devise.en.yml

```diff
@@ -30,13 +30,13 @@ en:
       updated: 'Your password was changed successfully. You are now signed in.'
       edit:
         change_password: "Change my password"
-        new_password: "New password"
-        confirm_password: "Confirm password"
+        new_password: "NEW PASSWORD"
+        confirm_password: "CONFIRM PASSWORD"
       new:
         forgot_password: "Forgot your password?"
         no_account: 'No account with this email exists'
         reset_password: "Reset password"
-        email: "Email address"
+        email: "EMAIL ADDRESS"
```

Comment:
Why are those changes necessary? We already have `text-tranform: uppercase` for the placeholders.